alphapulldown/scripts/run_structure_prediction.py

```diff
@@ -95,6 +97,42 @@
 # AlphaLink2 settings
 flags.DEFINE_string('crosslinks', None, 'Path to crosslink information pickle for AlphaLink.')
 
+# AlphaFold3 settings
+# JAX inference performance tuning.
+flags.DEFINE_string(
+    'jax_compilation_cache_dir',
+    None,
+    'Path to a directory for the JAX compilation cache.',
+)
+flags.DEFINE_list(
```

Comment:
What happens if someone specifies these options using another backened?

Reply:
they will be silently ignored